model/task/task.go

```diff
@@ -64,13 +64,17 @@ type Task struct {
 	// scheduled - the time the commit is scheduled
 	// start - the time the agent starts the task on the host after spinning it up
 	// finish - the time the task was completed on the remote host
-	CreateTime    time.Time `bson:"create_time" json:"create_time"`
-	IngestTime    time.Time `bson:"injest_time" json:"ingest_time"`
-	DispatchTime  time.Time `bson:"dispatch_time" json:"dispatch_time"`
-	ScheduledTime time.Time `bson:"scheduled_time" json:"scheduled_time"`
-	StartTime     time.Time `bson:"start_time" json:"start_time"`
-	FinishTime    time.Time `bson:"finish_time" json:"finish_time"`
-	ActivatedTime time.Time `bson:"activated_time" json:"activated_time"`
+	// activated - the time the task was marked as available to be scheduled, automatically or by a developer
+	// unblocked - for tasks that have dependencies, the time all dependencies are met
```

Comment:
It took me a while to understand this comment;  I would argue for keeping the name "UnblockedTime", or changing the bson/json tags; I don't totally understand why you need the tags to match a change that's been reverted. How many tasks actually have this unblocked_time field then? It might be okay to start fresh.

Separately, I would change the comment at 69 to say tags instead of db name, because db name implied to me the name of a database, which is confusing.

Reply:
That's a good point that these are tasks and so the new tasks aren't going to be having this name in the db, I'll just change it to match. "Unblocked" has a different usage elsewhere and I don't want to confuse that.